math/src/components/Calculator.css

```diff
@@ -0,0 +1,47 @@
+.container {
+  background: rgb(22, 20, 20);
+  top: 50%;
+  left: 50%;
+  right: 50%;
+  margin: 0 auto;
+  width: 300px;
+  text-align: center;
+  border: 1px solid black;
+  border-radius: 5px;
+}
+
+.keys {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: minmax(40px, auto);
```

Comment:
- [ ] Still, there is some issue with the design. You have added the extra button of `00` which is not required in this design. And also put the default value `0` on the display screen.

### Your design
![Screenshot from 2021-09-18 22-02-54](https://user-images.githubusercontent.com/22434660/133895914-d08a2dd3-ce9f-4a90-9248-d1b8cdd59f3c.png)


### Given design

![Screenshot from 2021-09-18 22-07-13](https://user-images.githubusercontent.com/22434660/133895949-8ac93752-3c11-4af2-a171-a611607db1dd.png)